.github/workflows/trivy_scan_before_release.yml

```diff
@@ -0,0 +1,49 @@
+name: Scan Theia Docker image for vulnerabilities before release
+
+on:
+  pull_request:
+    branches:
+      - release-*
+
+jobs:
+  build:
+    if: startsWith(github.event.pull_request.title, 'Release ')
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image:
+          - spark-jobs
+          - clickhouse-monitor
+          - clickhouse-server
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build theia Docker images
+      id: make
+      run: |
+        make ${{ matrix.image }}
+    - name: Run Trivy vulnerability scanner on latest Theia ${{ matrix.image }} image
+      if: ${{ always() && steps.make.conclusion == 'success' }}
+      uses: aquasecurity/trivy-action@0.10.0
+      continue-on-error: true
+      # we cannot use .trivy.yml as we need to override some config parameters
+      # and that is not supported by aquasecurity/trivy-action
```

Comment:
I think you meant to use that comment for the other workflow? For this one, you are actually using .trivy.yml for the config.